project/shared_factory/shared_factory.py

```diff
@@ -0,0 +1,13 @@
+import factory
+from contextlib import suppress
+from freezegun import freeze_time
+from functools import partial
+
+class FreezeTimeModelFactory(factory.django.
+DjangoModelFactory):
+
+    @classmethod
+    def create(cls, _fake_time=None, **kwargs):
+        wrapper = partial(freeze_time,time_to_freeze=_fake_time) if _fake_time else suppress
+        with wrapper(_fake_time):
+            return super().create(**kwargs)
```

Comment:
Make sure all files have an empty line at the end
```suggestion
            return super().create(**kwargs)
            
```